src/sql/base/browser/ui/table/tableDataView.ts

```diff
@@ -155,43 +151,46 @@ export class TableDataView<T extends Slick.SlickData> implements IDisposableData
 		this._onRowCountChange.fire(this.getLength());
 	}
 
-	find(exp: string, maxMatches: number = 0): Thenable<IFindPosition> {
+	find(exp: string, maxMatches?: number): Promise<IFindPosition> {
```

Comment:
It would be super awesome to have a test for this - it looks like we already have a test class for tableViewData so it shouldn't be difficult to add another test for find